@uportal/eyebrow-user-info/babel.config.js

```diff
@@ -2,6 +2,6 @@ module.exports = function(api) {
   api.cache.never();
   return {
     plugins: ['babel-plugin-transform-custom-element-classes'],
-    presets: [['@vue/app', {useBuiltIns: false}]],
+    presets: [['@vue/app', {useBuiltIns: 'usage'}]],
```

Comment:
:art: `useBuiltIns: 'usage'` is the default https://cli.vuejs.org/guide/browser-compatibility.html#polyfills
This could be simplified to `presets: ['@vue/app']`

Reply:
@ChristianMurphy : thx for  :eyes:  I rewrote my change fallowing your comment :wink: